README.md

```diff
@@ -356,17 +356,14 @@ Release to [clojars](https://clojars.org/):
 lein with-profile plugin.mranderson/config deploy clojars
 ```
 
-Or you can use the "very sophisticated" `build.sh` script to clean,
-inline, and run a lein target like `install` or `deploy`:
+Or you can use the Makefile way:
```

Comment:
`Makefile way` sounds a bit strange, maybe just `Makefile` or `Makefile as` or something like that?